layers/+distribution/spacemacs-base/packages.el

```diff
@@ -300,7 +300,7 @@ Example: (evil-map visual \"<\" \"<gv\")"
         i.e. `SPC m h h`, to lookup the source of the definition,
         while falling back to `evil-lookup'"
         (interactive)
-        (condition-case nil
+        (condition-case-unless-debug nil
```

Comment:
Actually this is just a poorly written function. It should be checking if the key binding exists directly rather than relying on some key sequence throwing an error.